content/blog/2018-05-10-onboarding-social-weather.md

```diff
@@ -0,0 +1,698 @@
+| \- i’m delighted that you find my work interesting and i’m very keen to help, contribute and collaborate in any capacity.                                                                                                                                                                                                                                         |   1.084493|
+| thank you very much for your thorough and thoughtful review, @batpigandme ! this is great feedback, and i think that visdat will be much improved because of these reviews.                                                                                                                                                                                       |   1.083653|
+| great, thank you very much for accepting this package. i am very grateful about the reviews, which were very helpful to improve this package!                                                                                                                                                                                                                     |   1.074281|
+| <br><br>                                                                                                                                                                                                                                                                                                                                                          |           |
+
+As you can imagine, these sentences make the whole team very happy! And
+we hope they’ll encourage you to contribute to rOpenSci onboarding.
+
+### Outlook
+
+This first try at text analysis is quite promising: we were able to
+retrieve text and to use natural language processing to extract most
+common words and bigrams, and sentiment. One could expand this analysis
+with a study of emoji use, *in* the text using an emoji dictionary [as
+in this blog post](http://www.masalmon.eu/2017/05/03/lucysemojis/) and
+*around* the text (so-called emoji reactions, present in the API and
+used in e.g
+[`ghrecipes::get_issues_thumbs`](https://github.com/maelle/ghrecipes/blob/master/R/get_issues_thumbs.R)).
+Besides, another aspect of social weather is maybe the timeliness that’s
+expected or implemented at the different parts of the process, but it’d
+be covered by other data such as the labelling history of the issues,
+which could get extracted from GitHub V4 API as well.
+
+This post is the final post of the “Our package review system in review”
+series. The first post presented [data collection from
+GitHub](https://ropensci.org/blog/2018/04/26/rectangling-onboarding/),
+the second post aimed at [quantifying the work represented by
+onboarding](https://ropensci.org/blog/2018/05/03/onboarding-is-work/).
+The posts comforted us in our efforts to try and automate more parts of
+the review system, and motivated us to keep using data to illustrate and
```

Comment:
In the Outlook, good to refer back to the term "social weather" and indicate what your data said about onboarding social weather. Maybe super-briefly summarize the nature of onboarding positive and negative terms to back up your claim that it is positive